app/controllers/application_controller.rb

```diff
@@ -85,6 +85,7 @@ def authenticate_token
     authenticate_with_http_token do |token, _options|
       @session = Session.find(token)
       return false if @session.nil?
+      Rails.logger.info("Logged in user with id #{@session.uuid}")
```

Comment:
I know that you can use a `proc` in log_tags to access the request - maybe we could tag part of a session or GA token from `request.cookie_jar` (or `anon` if there is none) to trace sessions, if that's a thing we need.